src/GNDStk/v1.9/transport/Reaction.hpp

```diff
@@ -88,27 +88,35 @@ class Reaction : public Component<Reaction> {
    // ------------------------
 
    // ENDF_MT
-   const auto &ENDF_MT() const
+   const Integer32 &
+   ENDF_MT() const
     { return content.ENDF_MT; }
-   auto &ENDF_MT()
+   Integer32 &
+   ENDF_MT()
```

Comment:
I'm confused here. When returning the Values node start value, the return types there were const Integer32 and Integer32 (without the reference). Is that because start needs to be treated differently from these (because they're BodyText related)?